python/pyspark/sql/utils.py

```diff
@@ -110,3 +110,12 @@ def toJArray(gateway, jtype, arr):
     for i in range(0, len(arr)):
         jarr[i] = arr[i]
     return jarr
+
+
+def _require_minimum_pyarrow_version():
```

Comment:
@ueshin did we do the same thing for pandas?

Reply:
No. I just checked if `ImportError` occurred or not. We should do the same thing for pandas later.